src/Test.EndToEnd.Baselining/Extensions/ArtifactUriEnumeration.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Sarif;
+
+namespace SarifBaseline.Extensions
+{
+    public static class ArtifactUriEnumeration
+    {
+        public static IEnumerable<Uri> AllArtifactUris(this SarifLog log)
+        {
+            if (log != null)
+            {
+                foreach (Run run in log.EnumerateRuns())
+                {
+                    foreach (Uri uri in run.AllArtifactUris())
```

Comment:
>AllArtifactUris [](start = 44, length = 15)

We might consider the GetXXX name convention for these cases. e.g., GetAllArtifactUris.

Reply:
Yep - I'll leave that for the "what will the SDK version look like?" design.